tests/models/animals/test_decay.py

```diff
@@ -0,0 +1,34 @@
+"""Test module for decay.py.
+
+This module tests the functionality of decay.py
+"""
+
+import pytest
+
+
+class TestCarcassPool:
+    """Test the CarcassPool class."""
+
+    def test_initialization(self):
+        """Testing initialization of CarcassPool."""
+        from virtual_rainforest.models.animals.decay import CarcassPool
+
+        c1 = CarcassPool(1000.7, 1)
+        assert pytest.approx(c1.stored_energy) == 1000.7
+
+
+class TestExcrementPool:
+    """Test the ExcrementPool class."""
+
+    def test_initialization(self):
+        """Testing initialization of CarcassPool."""
+        from virtual_rainforest.models.animals.decay import ExcrementPool
+
+        poo = ExcrementPool(77.7, 25.0)
```

Comment:
I've seen people name variables `poo` before, but I think this is the first time where it's actually relevant :stuck_out_tongue: